frontends/web/src/locales/en/app.json

```diff
@@ -756,6 +760,10 @@
         "text": "The fee is based on the transaction data size and not its amount. The fee targets are calculated by Bitcoin Core's fee estimation algorithm for each network priority you chose. They are shown if they have a different value from the target below.\nEconomy: 24 blocks (around 4 hours for Bitcoin, 1 hour for Litecoin)\nLow: 12 blocks (around 2 hours for Bitcoin, 30 minutes for Litecoin)\nNormal: 6 blocks (around 1 hour for Bitcoin, 15 minutes for Litecoin)\nHigh: 2 blocks (around 20 minutes for Bitcoin, 5 minutes for Litecoin)\n(A block takes on average ten minutes for Bitcoin (2.5 minutes in Litecoin) to mine and the network load may vary considerably in the above periods.)",
         "title": "How is the fee determined?"
       },
+      "plugout": {
+        "text": "No, once you have made a send transaction, you do not need to leave your BitBox plugged in. This is because the transaction has already been broadcasted to the blockchain and is on its way to the destination address. You are free to disconnect your BitBox.",
```

Comment:
same here, remove the middle sentence?

Reply:
Also 'Once you have made a transaction' instead of 'send transaction'? 'send transaction' sounds a bit confusing imho

Reply:
fixed in https://github.com/digitalbitbox/bitbox-wallet-app/compare/a9cc9deb8bca1a52da3b0843c9677ed8488529f2..a568f6a533009adc0997c43348819fc82f0fbb00